cs6232-g4/View/SearchFurnitureForm.cs

```diff
@@ -0,0 +1,111 @@
+﻿using Employees.Controller;
+using Furnitures.Controller;
+using Furnitures.Model;
+using System.Drawing.Text;
+using System.Windows.Forms;
+using System.Xml.Linq;
+
+/// <summary>
+/// Furniture Search Form
+/// Author: LM
+/// 30 Mar 2024
+/// </summary>
+/// 
+namespace cs6232_g4.View
+{
+    public partial class SearchFurnitureForm : Form
+    {
+        private readonly EmployeeController _employeeController;
+        private readonly FurnitureController _furnitureController;
+        private List<Furniture> _furniture;
+        private bool loggedOut;
+        private readonly LoginForm _loginForm;
+
+        public SearchFurnitureForm(LoginForm loginForm)
+        {
+            InitializeComponent();
+        }
+        private void Clear()
+        {
+            RentalComboBox.Text = "";
+            CategoryComboBox.Text = "";
+            StyleComboBox.Text = "";
+
+        }
+        private void SearchFurniture_FormClosing(object sender, FormClosingEventArgs e)
+        {
+            if (!loggedOut)
+            {
+                Application.Exit();
+            }
+        }
+        private void LogoutLink_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
+        {
+            loggedOut = true;
```

Comment:
the SearchFurniture form shouldn't be coupled with the Login form. Basically this is handled as part of the Dashboard form because the logic for login/logout is general for all tabs so it shouldn't be implemented in SearchFurniture tab but in the place general to all of them which is dashboard

Please remove all login related code and only worry about searching furniture

Reply:
The user must be logged in in order to view it, so it has to call some of the methods from login. I'm happy to have a discussion about this if it's helpful.